spring-apps/autosleep-core/src/main/java/org/cloudfoundry/autosleep/ui/web/model/OrgEnrollmentConfigRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.cloudfoundry.autosleep.ui.web.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class OrgEnrollmentConfigRequest {
+
+    @JsonProperty("idle-duration")
+    private String idleDuration;
+
+    @JsonProperty("exclude-spaces-from-auto-enrollment")
+    private String excludeSpacesFromAutoEnrollment;
+
+    @JsonProperty("state")
+    private String state;
+
+    @JsonProperty("auto-enrollment")
+    private String autoEnrollment;
```

Comment:
This enum is actually used for the model object. The request object was created to intercept the data from the call and perform full validation on it.
The refactoring of the forcedAutoEnrollment value to an enum will be pushed as part of this story https://www.pivotaltracker.com/story/show/134540625